packages/css-data/src/shorthands.test.ts

```diff
@@ -35,6 +35,23 @@ test("expand border", () => {
   ]);
 });
 
+test("expand border with css-wide keywords", () => {
+  expect(expandShorthands([["border", "INHERIT"]])).toEqual([
+    ["border-top-width", "INHERIT"],
```

Comment:
why is this one upper case?

Reply:
css is case-insensitive in most cases. Just a check it handles it.

Reply:
would be better to cheeck as a separate test, I wouldn't have guessed the test is about case sensitivity